manifests/server.pp

```diff
@@ -16,7 +16,7 @@
   Hash $databases                          = {},
   $ldap_ifs                                = ['/'],
   $ldaps_ifs                               = [],
-  $ldapi_ifs                               = ['/'],
+  $ldapi_ifs                               = undef,
```

Comment:
I'm wondering it wouldn't be more elegant to default to `[]` and check the size of the array in the condition.